goal_src/jak2/engine/debug/default-menu.gc

```diff
@@ -4127,10 +4125,11 @@
              (function "Hover Drillmid" #f (lambda ()
                                              (let ((a0-0 (get-nav-graph-editor)))
                                                (when a0-0
-                                                 (let ((t9-1 (method-of-object a0-0 nav-graph-editor-method-62)))
-                                                   'hover
+                                                 (let ((t9-1 (method-of-object a0-0 nav-graph-editor-method-62))
```

Comment:
Looks like `nav-graph-editor-method-62` takes one more argument (the `'drillmid`)